### PR TITLE
port `manta-germline` to nf-test

### LINKED ISSUE #5360: port `manta/germline` to nf-test

### PR DESCRIPTION
**Pull request by famosab**
## PR checklist

Closes #5360 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [ ] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

File: modules/nf-core/manta/germline/tests/main.nf.test.snap
Comment (nvnieuwk) [inline modules/nf-core/manta/germline/tests/main.nf.test.snap:1]: Can you regenerate the snapshot? It doesn't match what you have in your test file

**Review by nvnieuwk — CHANGES_REQUESTED** [review]

File: modules/nf-core/manta/germline/tests/main.nf.test.snap
Comment (famosab) [reply modules/nf-core/manta/germline/tests/main.nf.test.snap:1]: I am running nf-core modules test locally and its taking ages ... hopefully I can add it soon.

Comment (nvnieuwk) [reply modules/nf-core/manta/germline/tests/main.nf.test.snap:1]: Thank you for your effort, `manta` isn't the fastest tool indeed :grin:

Comment (famosab) [reply modules/nf-core/manta/germline/tests/main.nf.test.snap:1]: ok locally I run into errors with 

Test Process GATK4_SVCLUSTER

**Review by nvnieuwk — APPROVED** [review]

**Pull request merged**